src/python/libcgroup.pyx

```diff
@@ -372,6 +372,42 @@ cdef class Cgroup:
         if ret is not 0:
             raise RuntimeError("cgroup_create_scope failed: {}".format(ret))
 
+    def set_uid_gid(self, tasks_uid, tasks_gid, ctrl_uid, ctrl_gid):
+        """Set the desired owning uid/gid for the tasks file and the entire cgroup hierarchy
+
+        Arguments:
+        tasks_uid - uid that should own the tasks file
+        tasks_gid - gid that should own the tasks file
+        ctrl_uid - uid to recursively apply to the entire cgroup hierarchy
+        ctrl_gid - gid to recursively apply to the entire cgroup hierarchy
+
+        Note:
+        Does not modify the cgroup sysfs.  Does not read from the cgroup sysfs
+        """
+        ret = cgroup.cgroup_set_uid_gid(self._cgp, tasks_uid, tasks_gid, ctrl_uid, ctrl_gid)
+        if ret is not 0:
+            raise RuntimeError("cgroup_create_scope failed: {}".format(ret))
+
+    def set_permissions(self, dir_mode, ctrl_mode, task_mode):
+        """Set the permission bits on the cgroup
+
+        Arguments:
+        dir_mode - permissions to set on the cgroup directory
+        ctrl_mode - permissions to set on the files in the directory, except tasks
+        task_mode - permissions to set on the tasks file
+
+        Note:
+        Does not modify the cgroup sysfs.  Does not read from the cgroup sysfs.  Use the stat
+        module to populate the modes [1]
+
+        [1] https://docs.python.org/3/library/stat.html
```

Comment:
Is the comment about populate modes is not clear.  Is that intended for default values?

Reply:
I understand the comment now, after the test cases.